test/cryptography/sign.js

```diff
@@ -116,6 +122,25 @@ ${defaultSecondSignature}
 			});
 	});
 
+	describe('#writeVarIt', () => {
+		it('should calculate correct value for n < 253', () => {
+			return expect(writeVarInt(5)).to.be.eql(Buffer.from([5]));
+		});
+		it('should calculate correct value for n < 65536', () => {
+			return expect(writeVarInt(300)).to.be.eql(Buffer.from([253, 44, 1]));
+		});
+		it('should calculate correct value for n > 65536', () => {
+			return expect(writeVarInt(65537)).to.be.eql(Buffer.from([254, 1, 0, 1, 0]));
+		});
+	});
+
+	describe('#messageDigest', () => {
+		it('should create message digest', () => {
+			const msgBytes = messageDigest(defaultMessage);
+			return expect(msgBytes).to.be.eql(defaultMessageDigest);
+		});
```

Comment:
What about an empty message or one that's too long? Should it handle buffers?

Reply:
I'll add type check for `string` type. Also, I think that message length checker should be not in this function. And which message should be considered as a too long message? (what length)

Reply:
One where the length can't be encoded as a varint. I'm OK with that check being delegated to the varint function though.